src/navigation/index.tsx

```diff
@@ -23,7 +23,10 @@ const RootNavigation = () => {
         <RootStack.Screen name={AppRoute.APP} component={AppNavigation} />
       ) : (
         <>
-          <RootStack.Screen name={AuthRoute.SIGN_IN} component={HomeScreen} />
+          <RootStack.Screen
+            name={AuthRoute.SIGN_IN}
+            component={OnboardingNavigation}
```

Comment:
Onboarding should be within `app`-routes, not `auth`

Reply:
I just forgot to remove onborading from navigation, as we need LoginScreen here, I added it here just for testing